Sources/Impl/PinLayout+Coordinates.swift

```diff
@@ -328,33 +328,33 @@ extension PinLayout {
     }
     
     internal func validateWidth(_ width: CGFloat, context: Context) -> Bool {
-        if width < 0 {
+        guard width.isNormal && width >= 0  else {
```

Comment:
you can check just that width `> 0`, `isNormal` should handle case for greater than zero. (just a quick remark)